src/test/java/PositionTests.java

```diff
@@ -48,4 +59,17 @@ void testNewPosition(int row, int column, Class<Exception> expectedException) {
     void arePositionsAdjacent(@NotNull Position firstPosition,@NotNull Position secondPosition, boolean expectedAdjacent) {
         assertEquals(firstPosition.isPositionAdjacentTo(secondPosition), expectedAdjacent);
     }
+
+    @ParameterizedTest
+    @MethodSource("providePositionPairsForComparison")
+    void testPositionOrdering(Position first, Position second, int expectedResult) {
+        if (expectedResult == 1) {
+            assertTrue(first.compareTo(second) > 0);
+        } else if (expectedResult == -1) {
+            assertTrue(first.compareTo(second) < 0);
+        } else {
+            assertEquals(0, first.compareTo(second));
```

Comment:
I would do `assertTrue(first.compareTo(second) == 0)` in order to have the same structure for all the three cases. You can laso add `assertNotEquals(first, second)` for the first and the second case

Reply:
Correction suggested by IntelliJ, I don't know what's the reason for it